pkg/usm/tracer_classification_test.go

```diff
@@ -5,8 +5,8 @@
 
 //go:build linux_bpf || (windows && npm)
 
-// Package tracer contains implementation for NPM's tracer.
-package tracer
+// Package usm contains tests for USM
+package usm
```

Comment:
Any reason we couldn't put these under `pkg/network/usm` or `pkg/network/tracer/usm`? The latter since all of the files seem to be prefixed with `tracer_`.